lib/net/ldap.rb

```diff
@@ -309,7 +309,14 @@ class Net::LDAP
     :constructed => constructed,
   }
 
+  universal = {
```

Comment:
@mynameisrufus I agree 100% that the parsing behavior needs to be refactored.

I'm uncomfortable accepting this as is without a better understanding of the specific value we're adding here. This change doesn't align with my understanding of the `ExtendedResponse` mechanics. I don't want to accept something that makes it work but isn't clear why because it will likely have unintended/unaccounted for consequences.